src/utils/axios.ts

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+const BASE_URL: string = import.meta.env.VITE_BASE_URL;
+
+export const axiosClient = axios.create({
+  baseURL: BASE_URL,
+  timeout: 1000,
```

Comment:
1 second might be a bit low here ultimately - 5 or 10 seconds is a decent limit to set before giving up on the request.